### ISSUE TITLE
Check EXERCISM_PRETTIER_VERSION matches prettier in package.json

### ISSUE THREAD

**Issue by SleeplessByte** [exercism/javascript#1196]
`prettier` is configured in our workflows as `EXERCISM_PRETTIER_VERSION` (see #1195 for an example). It's version in our dependency list in `package.json` (currently `2.3.2`) must match the workflow setting exactly. Why? `prettier` will be installed if a maintainer is working on the track, and the workflow will check it against a version. If these don't match, a cycle of "updates" and "workflow failures" will occur.

In other words, given a `package.json` version of `2.3.2`, the ENV variable must be `2.3.2`.

Your task, should you accept it, is to add a check to the "precheck" scripts that run automatically on PR requests is to check if the package.json version matches the ENV variable or that the ENV variable is _set_ / _replaced_ with the `package.json` version.

**Comment by laulujan** [comment]
Hello!! Is it ok if I give it a try??

**Comment by SleeplessByte** [comment]
@laulujan go for it! Let me know / ping me if you need help.

**Comment by laulujan** [comment]
Hi @SleeplessByte I opened a PR #1204 on this issue, could you tell me if I am on a good track or how should I approach this?  I am also wondering how to test this, Thanks in advance!!
